src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/HtmlFormatter.cs

```diff
@@ -34,7 +34,7 @@ internal class HtmlFormatter
 
             var @params = new DocumentFormattingParams()
             {
-                TextDocument = new TextDocumentIdentifier { Uri = FilePathNormalizer.Instance.Normalize(context.Uri.GetAbsoluteOrUNCPath()) },
+                TextDocument = new TextDocumentIdentifier { Uri = context.Uri },
```

Comment:
Could you git blame this bit to ensure we're not accidentally re-introducing a bug via the FilePathNormalizer call?